go/arrow/ipc/ipc.go

```diff
@@ -104,6 +108,32 @@ func WithSchema(schema *arrow.Schema) Option {
 	}
 }
 
+// WithLz4 tells the writer to use LZ4 Frame compression on the data
+// buffers before writing. Requires >= Arrow 1.0.0 to read/decompress
+func WithLZ4() Option {
```

Comment:
small style nit:  Is there a go standard for LZ4 vs Lz4? (the documentation and method name are inconsistent here).

Reply:
The standard would be the way the function is named as `LZ4`, i'll fix the doc.